src/controller/credential_controller.ts

```diff
@@ -154,7 +156,7 @@ export async function issueVC(
       return res.json({ result: "VC not issued" });
     });
 
-    return res.status(200).json({id: cred.id})
+  return res.status(200).json({ id: cred.id, identifier: cred.identifier });
```

Comment:
Just a thought, should we send complete IDocument as a response here?

Reply:
I think identifier is enough, because we have a 'GET' call for that.